src/Tests/Microsoft.NET.Sdk.Razor.Tests/BuildIntegrationTest.cs

```diff
@@ -67,7 +67,7 @@ public void Build_SimpleMvc_NoopsWithRazorCompileOnBuild_False()
             new FileInfo(Path.Combine(outputPath, "SimpleMvc.Views.pdb")).Should().NotExist();
         }
 
-        [Fact]
+        [CoreMSBuildOnlyFact]
```

Comment:
Do you know why this is failing?

Reply:
No 😞  but I opened https://github.com/dotnet/aspnetcore/issues/33796 to dedicate some time to it. It seems to only happen for these tests and some Helix funkiness is preventing me from the other tests that invoke the iRSG but aren't failing with this error.